lib/telegraf/telegraf.conf.erb

```diff
@@ -572,7 +573,9 @@
   ##    root@tcp(127.0.0.1:3306)/?tls=false
   #
   ## If no servers are specified, then localhost is used as the host.
-  servers = ["root@tcp(127.0.0.1:3306)/"]
+  servers = <%= config_file['mysql'] %>
+  tagexclude = ["server"]
```

Comment:
I actually think we will want this in, or we will want to be sure we're setting some root namespace explicitly.  the reported name of ip:port is not as good as a real name, but I think it's something people will want in their metrics - the name of the server you're talking to is more useful than the name of the server you're talking from.